src/ecmult_const_impl.h

```diff
@@ -15,7 +15,8 @@
 /* This is like `ECMULT_TABLE_GET_GE` but is constant time */
 #define ECMULT_CONST_TABLE_GET_GE(r,pre,n,w) do { \
     int m; \
-    int abs_n = (n) * (((n) > 0) * 2 - 1); \
+    int mask = n >> (sizeof(n) * CHAR_BIT - 1); \
```

Comment:
Add parentheses around `n`? Also in the line below.

Reply:
doh. Thanks. done.